lib/include/lib/settings/general.hpp

```diff
@@ -129,6 +129,11 @@ namespace lib
 			 * Show a notification on track change
 			 */
 			bool notify_track_change = false;
+
+			/**
+			 * Hide to system tray instead of quitting when the close button is pressed
+			 */
+			bool hide_to_tray = false;
```

Comment:
Maybe "close to tray" is a better name? Feels like it's more obvious what it does, as it could be mixed up as a "minimize to tray". Also applies to all other references.

```suggestion
			bool close_to_tray = false;
```

Reply:
Alright, now this is `close_to_tray`, and other variables are consistent with this convention.